tox.ini

```diff
@@ -3,17 +3,13 @@ envlist = py{27,35}, py{27,35}-marionette, flake8
 
 [testenv]
 passenv = DISPLAY PYTEST_ADDOPTS
-deps =
-    mock
-    pytest==3.0.4
-    pytest-selenium
-    pytest-xdist==1.15.0
+deps = -rrequirements/tests.txt
 commands = pytest --ignore tests/marionette {posargs}
 
 [marionette]
 deps =
     {[testenv]deps}
-    marionette_driver==2.1.0
+    -rrequirements/marionette.txt
```

Comment:
Nice - I didn't know you could specify multiple requirements files. I was expecting that marionette.txt would simply import from tests.txt.